certified-connectors/Exasol/apiDefinition.swagger.json

```diff
@@ -0,0 +1,524 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This connector enables you to work with the Exasol Database.",
```

Comment:
@pj-spoelders Please provide a more detailed description that should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
Hope this is OK.